docs/src/aws-construct-lib.rst

```diff
@@ -103,9 +103,274 @@ were part of your app.
+
+In cases the strongly-typed overrides are not sufficient, or, for example, if
+the schema defined in CloudFormation is not up-to-date, the method
+:py:meth:`cdk.Resource.addOverride(path, value) <@aws-cdk/cdk.Resource.addOverride>`
+can be used to define an override that will by applied to the resource
+definition during synthesis.
+
+For example:
+
+.. code-block:: ts
+
+   // define an override at the resource definition root
+   bucketResource.addOverride('Transform', 'Boom');
+
+   // define an override for a property (both are equivalent operations):
+   bucketResource.addPropertyOverride('VersioningConfiguration.Status', 'NewStatus');
+   bucketResource.addOverride('Properties.VersioningConfiguration.Status', 'NewStatus');
+
+   // use dot-notation to define overrides in complex structures which will be merged
+   // with the values set by the higher-level construct
+   bucketResource.addPropertyOverride('LoggingConfiguration.DestinationBucketName', otherBucket.bucketName);
+
+   // it is also possible to assign a `null` value if this is your thing
+   bucketResource.addPropertyOverride('Foo.Bar', null);
+
+Will synthesize to:
+
+.. code-block:: json
+
+   {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+         "Foo": {
+            "Bar": null
+         },
+         "VersioningConfiguration": {
+            "Status": "NewStatus"
+         },
+         "LoggingConfiguration": {
+            "DestinationBucketName": {
+               "Ref": "Other34654A52"
+            }
+        }
+      },
+      "Transform": "Boom"
+   }
+
+Use ``undefined``, :py:meth:`cdk.Resource.addDeletionOverride <@aws-cdk/cdk.Resource.addDeletionOverride>`
+or :py:meth:`cdk.Resource.addPropertyDeletionOverride <@aws-cdk/cdk.Resource.addPropertyDeletionOverride>`
+to delete values:
+
+.. code-block:: ts
+
+   const bucket = new s3.Bucket(this, 'MyBucket', {
+      versioned: true,
+      encryption: s3.BucketEncryption.KmsManaged
+   });
+
+   const bucketResource = bucket.findChild('Resource') as s3.cloudformation.BucketResource;
+   bucketResource.addPropertyOverride('BucketEncryption.ServerSideEncryptionConfiguration.0.EncryptEverythingAndAlways', true);
+   bucketResource.addPropertyDeletionOverride('BucketEncryption.ServerSideEncryptionConfiguration.0.ServerSideEncryptionByDefault');
+
+Will synthesize to:
+
+.. code-block:: json
+
+   "MyBucketF68F3FF0": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+         "BucketEncryption": {
+            "ServerSideEncryptionConfiguration": [
+               {
+                  "EncryptEverythingAndAlways": true
+               }
+            ]
+         },
+         "VersioningConfiguration": {
+            "Status": "Enabled"
+         }
+      }
+   }
+
+Directly Defining CloudFormation Resources
+-------------------------------------------
+
+It is also possible to explicitly define CloudFormation resources in your stack.
+To that end, simply instantiate one of the constructs under the
```

Comment:
Remove "simply" everywhere as it is unnecessary.

Reply:
Fixed